nautobot/core/testing/filters.py

```diff
@@ -124,6 +124,8 @@ def test_boolean_filters_generic(self):
             for filter_name, filter_object in self.filterset.get_filters().items():
                 if not isinstance(filter_object, RelatedMembershipBooleanFilter):
                     continue
+                if filter_object.method is not None:
```

Comment:
Custom method e.g `has_interface_assignments` does not fit the default boolean logic here. So we ignore that and redefine it in the test case itself.